webapp/components/test-results-history-grid.html

```diff
@@ -96,6 +97,13 @@
         return params;
       }
 
+      subtestNameForDisplay(subtestName, isTestHarness) {
```

Comment:
This is redundant now that `AbstractTestFileResultsTable` is mixed in.

Reply:
Done.